VERSION.sh

```diff
@@ -12,7 +12,7 @@ VERSION_MAJOR=2
 VERSION_MINOR=2
 VERSION_MAINT=0
 # NOTE: VERSION_PREREL is automatically set for builds made out of a git tree
-VERSION_PREREL=0
+VERSION_PREREL=
 VERSION_DATE=$(date -u +%Y%m%d%H%M)
```

Comment:
Does this mess with the version number when doing "make install" or "make -f rpm.mk rpms"?

Reply:
Yes, it breaks `make -f rpm.mk rpms` build when `.git` directory is not present. For example, from tarball:
```
cp /workspace/ds/rpmbuild/RPMS/*/*2.2.00*.rpm dist/rpms/
cp: cannot stat '/workspace/ds/rpmbuild/RPMS/*/*2.2.00*.rpm': No such file or directory
make: *** [rpm.mk:140: rpms] Error 1
```

`VERSION_PREREL` should be empty for the real releases. And it is automatically populated with git commit hash if it's a development build. It was empty until b4d22b67463586561f4621dce4b3b198f3858373.

Reply:
The reason we started to see failures now: for quite some time CI images were not updated automatically, they had an older version of Git. I've updated CI images to F36, but they broke Compile action (this issue fixes it). I've reverted back to F35, but the updated image now contains new version of Git that has a new behaviour after CVE-2022-24765 was fixed (see https://github.blog/2022-04-12-git-security-vulnerability-announced/ and https://github.blog/2022-04-18-highlights-from-git-2-36/). This caused `git` commands not to be executed after checkout, for example, we use them to set `VERSION_PREREL` to the commit hash.
I'll open a new issue for this and move `VERSION_PREREL` change there. And this PR should be merged after.